src/streaming/MediaPlayer.js

```diff
@@ -1920,6 +1920,33 @@ function MediaPlayer() {
     }
 
     /**
+     * Get the value of useDeadTimeLatency in AbrController. @see setUseDeadTimeLatencyForAbr
+     *
+     * @returns {boolean=}
+     *
+     * @memberof module:MediaPlayer
+     * @instance
+     */
+    function getUseDeadTimeLatencyForAbr() {
+        return abrController.getUseDeadTimeLatency();
+    }
+
+    /**
+     * Set the value of useDeadTimeLatency in AbrController. If true, only the download
+     * portion will be considered part of the download bitrate and latency will be
+     * regarded as static. If false, the reciprocal of the whole transfer time will be used.
+     * Defaults to true.
+     *
+     * @param {boolean=} useDeadTimeLatency - True or false flag.
+     *
+     * @memberof module:MediaPlayer
+     * @instance
+     */
+    function setUseDeadTimeLatencyForAbr(useDeadTimeLatency) {
+        abrController.setUseDeadTimeLatency(useDeadTimeLatency);
+    }
```

Comment:
Yeah, this is a problem for all the things that are set on AbrController - Perhaps a better move is to store a json config object on MediaPlayer.js; this way we can be sure config has the same lifetime as the player. It would also  make the API on MediaPlayer look a bit nicer without the huge number of functions for turning flags on and off.